SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -11,31 +13,64 @@ DEFINE_LOG_CATEGORY(LogSpatialVirtualWorkerTranslator);
 SpatialVirtualWorkerTranslator::SpatialVirtualWorkerTranslator()
 	: NetDriver(nullptr)
 {
+	bWorkerEntityQueryInFlight = false;
+	bIsReady = false;
+	DesiredVirtualWorkerCount = 0;
 }
 
 void SpatialVirtualWorkerTranslator::Init(USpatialNetDriver* InNetDriver)
 {
 	NetDriver = InNetDriver;
+	if (NetDriver != nullptr)
```

Comment:
This section could be a ternary but I don't really mind. Whatever you think is best.

Reply:
A repo style thing is to be explicit for nullptr.

Reply:
I seem to remember reading that UnrealGDK preferered the explicit check, otherwise I'm happy to change it. Matt Y?

Also, updated to use a ternary because you're right it's more readable in this case.

Reply:
Noted